ndcube/utils/wcs.py

```diff
@@ -207,7 +207,7 @@ def _wcs_slicer(wcs, missing_axes, item):
             item_ = _slice_list(item_checked)
             new_wcs = wcs.slice(item_)
             for i, it in enumerate(item_checked):
-                if isinstance(it, int):
+                if isinstance(it, (int, np.int64, np.int32)):
```

Comment:
I *think* this will work
```suggestion
                if isinstance(it, numbers.Integral):
```

Reply:
Looks like it does!  Thanks!